src/tests/App.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+
+
+export default (styled: Function) => {
+  const App = styled('div', {
+    margin: '50px',
```

Comment:
please rely in all examples on jss-preset-default, in this case it allows you to use numbers